commercetools-models/src/main/java/io/sphere/sdk/orders/OrderImportDraftImpl.java

```diff
@@ -157,7 +157,7 @@ public Address getShippingAddress() {
 
     @Override
     @Nullable
-    public OrderShippingInfo getShippingInfo() {
+    public ShippingInfoImportDraft getShippingInfo() {
```

Comment:
I didn't see this mentioned in our release notes?

Reply:
actually this class has default visibility so it shouldn't be documented.